src/linagora.esn.calendar/app/core/calendar-date-indicator.directive.js

```diff
@@ -1,4 +1,4 @@
-(function() {
+(function(angular) {
   'use strict';
```

Comment:
missing require for `CAL_EVENTS` constant and `calendarCurrentView` service locations.